app/routers/event.py

```diff
@@ -19,3 +24,73 @@ async def eventedit(request: Request):
 async def eventview(request: Request, id: int):
     return templates.TemplateResponse("event/eventview.html",
                                       {"request": request, "event_id": id})
+
+
+def get_event_by_id(db: Session, event_id: int) -> Event:
+    """Select event by id"""
+    return db.query(Event).filter(Event.id == event_id).first()
+
+
+# copy from michael ben david
+def check_validation(start_time: datetime, end_time: datetime) -> bool:
+    """Check if the start time is earlier than the end time"""
+
+    if start_time < end_time:
+        return True
+    return False
+
+
+def get_column_names_event(db: Session) -> Set:
+    result = db.execute("select * from events")
```

Comment:
Prefer using the ORM instead.

Also, If possible, try to get the column names without selecting the whole table, which might be a very heavy query.